pkg/kwokctl/runtime/binary/cluster.go

```diff
@@ -742,6 +744,61 @@ func (c *Cluster) LogsFollow(ctx context.Context, name string, out io.Writer) er
 	return nil
 }
 
+// CollectLogs returns the logs of the specified component.
+func (c *Cluster) CollectLogs(ctx context.Context, name string, dir string) error {
+	logger := log.FromContext(ctx)
+	var errs []error
+
+	path := filepath.Join(dir, consts.RuntimeTypeBinary+"-info.txt")
+	f, err := os.Create(path)
+	if err != nil {
+		errs = append(errs, err)
```

Comment:
```suggestion
		return err
```

If the first file fails to be created I don't think it's necessary to continue